supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -575,7 +577,28 @@ class PipelinesApiAsyncBackendJobExecutionActor(override val standardParams: Sta
     }
   }
 
+  var lastStatusPoll: Option[OffsetDateTime] = None
+  val junkMetadataValue = "Blah blah blah what a load of junk"
+  val configuredSpamQuantity = standardParams.configurationDescriptor.backendConfig.as[Option[Long]]("metadata_spam_per_second_on_poll")
+
+  def spamJunkMetadata() = {
+    configuredSpamQuantity foreach { quantity =>
+      val junkToSend = lastStatusPoll match {
+        case None =>
+          quantity
+        case Some(previousPoll) =>
+          val timeDiffSeconds = OffsetDateTime.now.toEpochSecond - previousPoll.toEpochSecond
+          timeDiffSeconds * quantity
+      }
+
+      lastStatusPoll = Option(OffsetDateTime.now)
+      val metadataJunk = 0.to(junkToSend.intValue) map { i => s"junk_value[$i]" -> junkMetadataValue }
```

Comment:
as discussed in Slack this may produce key collisions but that *probably* shouldn't matter (no uniqueness constraint and the summarizer is supposed to squash those duplicates)